apollo-core/src/main/java/com/ctrip/framework/foundation/internals/NetworkInterfaceManager.java

```diff
@@ -112,22 +113,15 @@ private void load() {
         m_local = InetAddress.getByName(ip);
         return;
       } catch (Exception e) {
-        System.err.println(e);
```

Comment:
I'm thinking maybe we need to keep this error message as this ip is specified by user with system property.

Reply:
ok, better to keep.